pkg/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -67,7 +66,7 @@ var (
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileClusterLogging) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	logrus.Debugf("Clusterlogging reconcile request.Name: '%s'", request.Name)
+	log.Info("Clusterlogging reconcile request.", "Name", request.Name)
```

Comment:
This should remain a debug level message, maybe v2?  or v3?

Reply:
done